protected/controllers/PatientController.php

```diff
@@ -1720,8 +1761,14 @@ public function actionUpdate($id)
 
             // not to be sync with PAS
             $patient->is_local = 1;
+            if (isset($_POST['PatientUserReferral'])) {
```

Comment:
Why are these two different if statements? Can't they just be two conditions in the same if?